src/main/java/com/graphaware/module/GraphDatabaseServiceAware.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2013-2016 GraphAware
+ *
+ * This file is part of the GraphAware Framework.
+ *
+ * GraphAware Framework is free software: you can redistribute it and/or modify it under the terms of
+ * the GNU General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details. You should have received a copy of
+ * the GNU General Public License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/>.
+ */
+package com.graphaware.module;
```

Comment:
My thinking, as you correctly guessed, what that nothing about the interface was specific to UUID and that it could perhaps see usage outside of the UUID module. In hindsight, as you point out, the chances of this are small. I'll relocate this interface into the .uuid package.